lib/utilities/validateRange.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Validates numbers ranges to not exceed 100 entities
+ *
+ * @param {*} value  The value to validate
+ * @param {*} helpers The helpers object
+ * @returns {Object} The value if validation passes
+ */
+export const validateRange = (value, helpers) => {
+    const MAX_RANGE_SIZE = 100;
+
+    const numbers = value.split(',').map((runNumber) => runNumber.trim());
```

Comment:
Good idea on extracting the code from RunFilter to a util function so that you can reuse it in LHCFilter but please pay attention to properly refactor the method. 
We are now testing general ranges and not runNumbers specifically